Improve contributing docs (#747)
FerretDB/FerretDB: .github/pull_request_template.md

# Checklist

> * [ ] I read [CONTRIBUTING.md](../CONTRIBUTING.md).

Reviewer: I would suggest adding here some links / highlites from contributing guidelines also.

> * [ ] I read [CONTRIBUTING.md](../CONTRIBUTING.md).

Reviewer 2: ```suggestion
* [ ] I've read [CONTRIBUTING.md](../CONTRIBUTING.md). I follow the guidelines provided there.
```

> * [ ] Tests are added for new functionality or fixed bugs.

Reviewer 2: ```suggestion
* [ ] I added tests for the new functionality or fixed bugs.
```

Author: My idea was to use exactly the same wording there and in CONTRIBUTING.md to make it easier to copy&paste

Reviewer 2: @AlekSi well, my concern is that for the first sentence you used active voice, and for the rest of the items - passive voice. So, there wasn't enough consistency there. Also, I believe, active voice is recommended when you implement a "call for action". Anyways, I leave it up to you.

> * [ ] `task all` passes.

Reviewer 2: ```suggestion
* [ ] I called `task all` locally, and it passed.
```
